src/components/Logo.tsx

```diff
@@ -0,0 +1,66 @@
+import { ComponentPropsWithRef, HTMLAttributes, forwardRef } from 'react'
+
+import PluralLogoFull from './icons/logo/PluralLogoFull'
+import PluralLogoMark from './icons/logo/PluralLogoMark'
+import PluralLogoWord from './icons/logo/PluralLogoWord'
+
+export type LogoProps = ComponentPropsWithRef<'div'> & {
+  isDark?: boolean
+  width?: string | number;
+  height?: string | number;
+  type?: LogoType;
+}
+
+export enum LogoType {
+  Full = 'full',
+  Word = 'word',
+  Mark = 'mark',
+}
+
+const Logo = forwardRef<HTMLDivElement, LogoProps & HTMLAttributes<HTMLDivElement>>(({
+  isDark = false,
+  width,
+  height,
```

Comment:
Sorry for nit-picking but I think we may simplify it a bit more.

You can default these to `= null` if that is needed and then in the code do the following: 

```js
if (!width && !height) {
  width = 320
  height = 79
}
```

With that you can set `width` and `height` for all variants without additional ifs. Would that work or I am missing something?